provider/junit5spring/src/main/kotlin/au/com/dius/pact/provider/spring/junit5/MockMvcTestTarget.kt

```diff
@@ -96,7 +97,11 @@ class MockMvcTestTarget @JvmOverloads constructor(
           val contentDisposition = ContentDisposition(bodyPart.getHeader("Content-Disposition").first())
           val name = StringUtils.defaultString(contentDisposition.getParameter("name"), "file")
           val filename = contentDisposition.getParameter("filename").orEmpty()
-          multipartRequest.file(MockMultipartFile(name, filename, bodyPart.contentType, bodyPart.inputStream))
+          if (filename.isEmpty()) {
+            multipartRequest.param(name, FileCopyUtils.copyToString(bodyPart.inputStream.bufferedReader()))
```

Comment:
Tbh it should be:
```suggestion
            multipartRequest.part(MockPart(name, FileCopyUtils.copyToByteArray(bodyPart.inputStream)))
```

But there is a problem with parsing parts in `MockMvc` in used version. It is already fixed (https://github.com/spring-projects/spring-framework/issues/26400) but requires library to be updated.

Reply:
I have upgraded the Spring libraries to 5.3.9